src/restaurant_inspections_api/constants.clj

```diff
@@ -6,3 +6,4 @@
 (def production? (env/production?))
 (def db-url (env/get-env-db-url))
 (def csv-files (env/get-csv-files))
+(def *earliest-date* "2013-01-01")
```

Comment:
what? no! sorry.... Imported convention from CLISP, earmuffs for global defs used as constants. What is the convention in Clojure?